src/inject/dynamic-theme/css-rules.ts

```diff
@@ -62,15 +62,18 @@ export function iterateCSSDeclarations(style: CSSStyleDeclaration, iterate: (pro
         }
         iterate(property, value);
     });
-    if (isSafari && style.cssText.includes('var(')) {
-        // Safari doesn't show shorthand properties' values
-        shorthandVarDepPropRegexps.forEach(([prop, regexp]) => {
-            const match = style.cssText.match(regexp);
-            if (match && match[1]) {
-                const val = match[1].trim();
-                iterate(prop, val);
-            }
-        });
+    if (isSafari) {
+        const cssText = style.cssText;
+        if (cssText.includes('var(')) {
+            // Safari doesn't show shorthand properties' values
+            shorthandVarDepPropRegexps.forEach(([prop, regexp]) => {
+                const match = cssText.match(regexp);
+                if (match && match[1]) {
+                    const val = match[1].trim();
+                    iterate(prop, val);
+                }
+            });
+        }
     } else {
         shorthandVarDependantProperties.forEach((prop) => {
```

Comment:
Hmm, so, what have you decided, to use `if (cssText.includes('var(')) {` for every browser or not?

Reply:
Hmm, not sure yet. Really depends how this code actually shall behave.

Reply:
It's a good performance for gmail so let's include it again.